strongbox-security/strongbox-user-management/src/test/java/org/carlspring/strongbox/users/service/AuthorizationConfigServiceTest.java

```diff
@@ -19,6 +20,7 @@
  * @author Alex Oreshkevich
  */
 @RunWith(SpringJUnit4ClassRunner.class)
+@ActiveProfiles(profiles = "test")
```

Comment:
Excuse my (apparent) ignorance, but why are you adding an active profile here for these tests? Do you want to run them in a suite, or what? I'm not sure I understand...?

Reply:
with this profile we can configure specific Spring environment just for unit tests (see [@Profile](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/context/annotation/Profile.html) annotation).